modules/ROOT/pages/_partials/upgrade-tool.adoc

```diff
@@ -0,0 +1,82 @@
+//This content is reused in:
+//mule-upgrade-tool.adoc
+
+// Before You Begin
+// tag::BeforeYouBegin[]
+
+* Ensure that you have xref:release-notes::mule-upgrade-tool/mule-upgrade-tool.adoc[the latest available version of Mule upgrade tool] so that the tool runs with the latest fixes and security enhancements.
+* A currently operational Mule instance running version 4.1.1 or later, in _stopped_ status to prepare for the upgrade.
++
+When updating your current Mule instance from a version between 4.1.1 and 4.1.4, use any patch update released after January 20, 2022. If the running version is not one of the previously mentioned, an error message appears stating that the Mule version cannot be upgraded unless an upgrade is made.
```

Comment:
Sorry, I still don't get this. 

Are they updating to a patch released after Jan 20, 2022?

Reply:
And I don't know what this means: "If the running version is not one of the previously mentioned, an error message appears stating that the Mule version cannot be upgraded unless an upgrade is made."